master/usage/migrating-pools.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Changing IP pools
+---
+
+## About changing IP pools
+
+When using {{ site.prodname }} IPAM, each workload is assigned an address from the selection of configured IP pools.
+You may want to modify the IP pool of a running cluster for one of the following reasons:
+- To move to a larger CIDR that can accommodate more workloads.
+- To move off of a CIDR that was used accidentally.
+
+### Purpose of this page
+
+Provide guidance on how to change from one IP pool to another on a running cluster.
+
+## Prerequisites
+
+**{{site.prodname}} IPAM**
+
+This guide only applies if you are using {{site.prodname}} IPAM.
+
+**Orchestrator support**
+
+While {{ site.prodname }} supports changing IP pools, not all orchestrators do.
+Be sure to consult the documentation of the orchestrator you are using to ensure it supports changing the workload CIDR.
+
+For example, in Kubernetes, all three of the following arguments must be equal to, or contain, the Calico IP pool CIDRs:
+
+- kube-apiserver: `--pod-network-cidr`
+- kube-proxy: `--cluster-cidr`
+- kube-controller-manager: `--cluster-cidr`
+
+OpenShift does not support changing the pod network CIDR (as per their [documentation on the `osm_cluster_network_cidr` configuration field](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-cluster-variables).
+
+**Application availability impact**
+
+This process will require the recreation of all {{ site.prodname }}-networked workloads, which will have some impact on
+the availability of your applications.
+
+## Consequences of deleting an IP pool without following this migration procedure
+
+Removing an IP pool without following this migration procedure can cause network connectivity disruptions in any running
+workloads with addresses from that IP pool. Namely:
+
+- If IP-in-IP was enabled on the IP pool, those workloads will no longer have their traffic encapsulated.
+- If nat-outgoing was enabled on the IP pool, those workloads will no longer have their traffic NAT'd.
+- If using Calico BGP routing, routes to pods will no longer be aggregated.
+
+## Changing an IP pool
+
+The basic process is as follows:
+
+1. Add a new IP pool.
+2. Disable the old IP pool. This prevents new IPAM allocations from the old IP pool without affecting the networking of existing workloads.
+3. Recreate all existing workloads that were assigned an address from the old IP pool.
+4. Remove the old IP pool.
+
+### Example: Kubernetes
+
+In this example, we created a cluster with kubeadm.  We wanted the pods to use IPs in the range
+`10.0.0.0/16` so we set `--pod-network-cidr=10.0.0.0/16` when running `kubeadm init`.  However, we
+installed {{ site.prodname }} without setting  default IP pool to match. Running `calicoctl get ippool -o wide` shows
```

Comment:
"without setting  default IP pool" - I think you've got an extra space where you want a "the"